### PR TITLE
Added default shard size setting

### PR DESCRIPTION
**Pull request by Anton-Shutik**
Added default shard size config

### CODE REVIEW

File: sprinklers/app_settings.py [hunk 1]
```diff
@@ -0,0 +1 @@
+SPRINKLER_DEFAULT_SHARD_SIZE = getattr(settings, 'SPRINKLER_DEFAULT_SHARD_SIZE', 20000)
```
Comment (al-the-x) [inline sprinklers/app_settings.py:4]: Settings `import`-ed where?

File: sprinklers/base.py [hunk 2]
```diff
@@ -125,8 +125,9 @@ def __repr__(self):
     def log(self, msg):
         logger.info("SPRINKLER %s: %s" % (self, msg))
 
+
 class ShardedSprinkler(SprinklerBase):
-    shard_size = 20000
+    shard_size = app_settings.SPRINKLER_DEFAULT_SHARD_SIZE
```
Comment (al-the-x) [inline sprinklers/base.py:131]: Same for `app_settings`...?

**Review by al-the-x — CHANGES_REQUESTED** [review]

File: sprinklers/base.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import app_settings
```
Comment (al-the-x) [inline sprinklers/base.py]: Use relative imports for clarity:
```suggestion
from . import app_settings
```

**Comment by al-the-x** [comment]
Can you link to the PR in `groveco/grove` that integrates this?

**Comment by Anton-Shutik** [comment]
@al-the-x https://github.com/groveco/grove/pull/7108 It's draft PR, because this code is not merged into main repo

**Review by al-the-x — APPROVED** [review]

**Pull request merged**